community/weave-scope/values.yaml

```diff
@@ -0,0 +1,80 @@
+# Where defaults exist, the values are set to them here.
+# Values with no preferred or common defaults are set to empty strings.
+global:
+  # global.image: the image that will be used for this release
+  image:
+    repository: quay.io/weaveworks/scope
+    tag: 1.10.1
+    # global.image.pullPolicy: must be Always, IfNotPresent, or Never
+    pullPolicy: "IfNotPresent"
+  # global.service.*: the configuration of the service used to access the frontend
+  service:
+    # global.service.name: the short name desired for the frontend service
+    # global.service.name may be specified if you need to use a specific service name, but will be generated if not specified
+    # global.service.name is a global so we can access its value easily from the agent subchart
+    # name: "weave-scope-app"
+    # global.service.port: (required if frontend.enabled == true) the port exposed by the Scope frontend service
+    # global.service.port is a global so we can access its value easily from the agent subchart
+    port: 80
+    # global.service.type: (required if frontend.enabled == true) the type of the frontend service -- must be ClusterIP, NodePort or LoadBalancer
+    # global.service.type is a global to keep it with the other values for configuring the frontend service
+    type: "ClusterIP"
+  # global.basicAuth.*: the configuration of the basic authentication
+  basicAuth:
+    enabled: false
+    username: ""
+    password: ""
```

Comment:
Passwords (or any sensitive data) should really not be entered via values.yaml as it is not secure.  (Any user with permissions to helm can access the helm release manifest and obtain sensitive info).  A better approach is have the user manually pre-create a Secret and then pass in the secret name.